client/tailscale/localclient.go

```diff
@@ -1418,25 +1418,25 @@ func (lc *LocalClient) CheckUpdate(ctx context.Context) (*tailcfg.ClientVersion,
 	return &cv, nil
 }
 
-// TailFSSetFileServerAddr instructs TailFS to use the server at addr to access
+// DriveSetFileServerAddr instructs drive to use the server at addr to access
```

Comment:
drop File in FileServer?

Drive not capitalized? If it's not a proper noun then "instructs drive to use" is weird lowercase.

Reply:
My preference would be to use `drive/Drive` for variable and package names, but in comments to refer to it as `Taildrive`. That's what I did [here](https://github.com/tailscale/corp/pull/18871).